cmd/print-follow-graph/print-follow-graph.go

```diff
@@ -0,0 +1,77 @@
+// Prints out a follow graph, as seen from the point of view of the focused peers.
+// This command is literally a pared down version of generate-test.go
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/ssb-ngi-pointer/netsim/generation"
+	"log"
+	"math/rand"
+	"path"
+	"sort"
+)
+
+func check(err error) {
+	if err != nil {
+		log.Fatalln(err)
+	}
+}
+
+func main() {
+	var args generation.Args
+	flag.StringVar(&args.FixturesRoot, "fixtures", "./fixtures-output", "root folder containing spliced out ssb-fixtures")
+	flag.IntVar(&args.MaxHops, "hops", 2, "the max hops count to use")
+	flag.IntVar(&args.FocusedCount, "focused", 2, "number of puppets to use for focus group (i.e. # of puppets that verify they are replicating others)")
+	flag.Parse()
+
+	var err error
+	g := generation.Generator{Args: args}
+	// map of id -> [list of followed ids]
+	var followMap map[string][]string
+	followMap, _, err = generation.GetFollowMap(path.Join(args.FixturesRoot, "follow-graph.json"))
```

Comment:
will do, thanks!

Reply:
ish...

the nice thing about returing type is that static analysis can pick it up and show you the comments along with it.

Reply:
i see :) well, i will keep it in mind for the future. right now i don't want to propagate that change across the different packages and commands, especially since my vim install doesn't pick up on showing the comments ;)

btw: am i right that in some cases i might have to do a type assertion if i wanted to use the named type?

Reply:
>  i might have to do a type assertion if i wanted to use the named type?

not in all circumstances. I need to follow up with the language spec actually.

btw: another interesting case of static analysis is godoc/pkg.go.dev